components/raft_log_engine/src/engine.rs

```diff
@@ -563,12 +563,17 @@ impl RaftEngine for RaftLogEngine {
         Ok(self.0.get_used_size() as u64)
     }
 
-    fn for_each_raft_group<E, F>(&self, _f: &mut F) -> std::result::Result<(), E>
+    fn for_each_raft_group<E, F>(&self, f: &mut F) -> std::result::Result<(), E>
     where
         F: FnMut(u64) -> std::result::Result<(), E>,
         E: From<engine_traits::Error>,
     {
-        unimplemented!()
+        for id in self.0.raft_groups() {
+            if id != STORE_REGION_ID {
```

Comment:
name `STORE_REGION_ID` is confusing

Reply:
What's your suggestion?

Reply:
FAKE_REGION_ID?

Reply:
It doesn't represent that it will be counted as global state.

Reply:
how about GLOBAL_STATE_FAKE_REGION_ID

Reply:
I rename it to `STORE_STATE_ID` as it is used to persist store level states.